ansible/roles/ocp4-workload-ccnrd/tasks/workload.yml

```diff
@@ -25,6 +25,26 @@
   debug:
     msg: "selected modules list: {{ modules }}"
 
+- name: Create ccn-redhat-operators CatalogSource
+  k8s:
+    state: present
+    merge_type:
+    - strategic-merge
+    - merge
+    definition: "{{ lookup('file', item ) | from_yaml }}"
+  loop:
+  - ./files/ccn-redhat-operators-cs.yaml
+
```

Comment:
Why not add the second file,to,the loop? :-)

Reply:
Good catch. I created a new PR: https://github.com/redhat-cop/agnosticd/pull/3149
Thanks!